handler.go

```diff
@@ -256,7 +256,10 @@ func (ca *Cagent) GetAllMeasurements() (MeasurementsMap, error) {
 
 	measurements = measurements.AddWithPrefix("services.", servicesList)
 
-	containersList, err := docker.ListContainers()
+	if ca.docker == nil {
```

Comment:
shouldn't we do that on `cagent` creation?

Reply:
done